rom/laghos_rom.cpp

```diff
@@ -2359,16 +2736,29 @@ ROM_Operator::ROM_Operator(ROM_Options const& input, ROM_Basis *b,
         ComputeReducedMe();
     }
 
-    if (hyperreduce && hyperreductionSamplingType == eqp)
-    {
-        // TODO: are reduced mass matrices needed for EQP, or just W matrices?
+	if (hyperreduce && hyperreductionSamplingType == eqp)
+	{
+		// basic EQP
+		// TODO: are reduced mass matrices needed for EQP, or just W matrices?
 
-        ComputeReducedMv();
-        ComputeReducedMe();
+		// TODO: why compute the reduced mass matrices, when for the basic EQP
+		// case we've already multiplied my M^{-1} and basis^T when deriving
+		// the reduced quadrature rules?
```

Comment:
Note my question here about computing the reduced mass matrices in the basic EQP case.